sample-app/helpers/Platform.tsx

```diff
@@ -0,0 +1,6 @@
+import { Platform } from 'react-native';
+
+export class PlatformType {
```

Comment:
why not just functions? there's no need for this class
```
const isIOS = (Platform.OS == 'ios');
const isAndroid = (Platform.OS == 'android');
```

and we'll have shorter calls instead of `PlatformType.isIOS ` -> `isIOS()`

Reply:
Outdated :/